### DISCUSSION TITLE
Issue with @expo/vector-icons in monorepo

### DISCUSSION THREAD

**Discussion by Puneet56** [nandorojo/solito#370] [General]
Hi, I am trying to get icons to work in with-tailwind monorepo. Added next-fonts which injects a webpack loader but still no luck. Working fine in older monorepo.

My next config

![image](https://user-images.githubusercontent.com/26213419/230288065-e44e4113-6524-4ac8-84a9-aae1759db63f.png)

Error
![image](https://user-images.githubusercontent.com/26213419/230287853-ee270f26-fe24-4015-8fcd-51703965748d.png)

**Comment by albertcito** [comment]
Did you find a solution? The last [comment](https://github.com/GeekyAnts/nativebase-templates/issues/43#issuecomment-1688344364) says add it to `transpileModules`. 
 
```javascript
const { withExpo } = require('@expo/next-adapter')

/** @type {import('next').NextConfig} */
const nextConfig = {
  ...
  reactStrictMode: false,
  transpilePackages: [
    'react-native',
    'react-native-web',
    'solito',
    'dripsy',
    '@dripsy/core',
    'moti',
    'app',
    'react-native-reanimated',
    '@expo/html-elements',
    '@expo/vector-icons',
    'react-native-gesture-handler',
  ],
}

module.exports = withExpo(nextConfig)
```

I did it, but in any case it doesn't work. I'm getting this issue:

```
../../node_modules/@expo/vector-icons/build/vendor/react-native-vector-icons/Fonts/Ionicons.ttf
Module parse failed: Unexpected character '' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)
```


I didn't tested it yet. But solito has a [Icon section](https://solito.dev/recipes/icons) in the documentation
